app/views/LocationShare/services/mapProviders.ts

```
@@ -0,0 +1,198 @@
+	provider: MapProviderName,
+	coords: Coords,
+	opts: StaticOpts = {}
+): { url: string; width: number; height: number } {
+	switch (provider) {
+		case 'google':
+			return buildGoogleStaticUrl(coords, opts);
+		case 'osm':
+			return buildOsmStaticUrl(coords, opts);
+		default:
+			// Exhaustiveness check
+			const _never: never = provider;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)